bridge/engine/SimulcastStream.h

```diff
@@ -50,6 +50,19 @@ struct SimulcastStream
     bool isSendingVideo() const { return _numLevels > 0 && _contentType == VideoContentType::VIDEO; }
     bool isSendingSlides() const { return _numLevels > 0 && _contentType == VideoContentType::SLIDES; }
 
+    uint32_t getLevelOf(uint32_t ssrc) const
+    {
+        for (size_t i = 0; i < _numLevels; ++i)
+        {
+            if (_levels[i]._ssrc == ssrc)
+            {
+                return i;
+            }
+        }
+
+        return 0;
```

Comment:
This zero sounds error prone. imo it could be an optional or change return type to `int32_t` and return `-1`